ReactNativeClient/lib/string-utils.js

```diff
@@ -122,7 +122,26 @@ function wrap(text, indent, width) {
 	});
 }
 
-function splitCommandString(command) {
+async function prepareCmdStringWindows(cmd, fnExists) {
+	// in windows, paths often contain spaces, so
+	// if there are spaces and the entire string points to an existing file,
+	// it's probably something like C:\Program Files\Foo\Foo.exe
+	if (cmd.indexOf(" ") != -1 && cmd.indexOf('"') == -1 && await fnExists(cmd)) {
+		cmd = '"' + cmd + '"';
+	}
+
+	// in windows, \ usually is a path delim, not an escape sequence
+	cmd = cmd.replace(/\\/g, '\\\\');
```

Comment:
Is there any reason we still need this, since in splitCommandString you are now disabling escape?

Reply:
Thanks, good catch.